optimize_sidebars.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  Components: [
+    {
+      type: "link",
+      label: "Overview Components",
+      href: "/docs/next/components",
+    },
```

Comment:
This is a foreseen down-side to the multi-version docs -- [having to rewrite the sidebars files, and duplicate them, due to the two plugins not knowing about each others' docs](https://github.com/camunda/camunda-platform-docs/pull/906#discussion_r874097861).